tools/tests/tooltip/test.js

```diff
@@ -31,6 +31,24 @@ function removeTooltips() {
     }
 }
 
+async function checkResizeHandler(hide = false, expected = true) {
+    const sandbox = sinon.createSandbox();
+    const tooltip = document.querySelector('#smart-position');
+    sandbox.spy(tooltip, "resizeDebounced");
+
+    await tooltip.show();
+    if (hide) tooltip.hide();
+
+

```

Comment:
remove one of these lines (I can't 'request change' here :D)

Reply:
Removed.